plugin.xml

```diff
@@ -24,7 +24,7 @@ xmlns:android="http://schemas.android.com/apk/res/android"
                 <source url="https://github.com/CocoaPods/Specs.git"/>
             </config>
             <pods>
-                <pod name="Firebase/InAppMessagingDisplay" spec="0.15.2" />
+                <pod name="Firebase/InAppMessagingDisplay" spec="~> 6.3.0" />
```

Comment:
@brotich to be sure, the version 6.3.0 refers to version of firebase in `package.json`?

Reply:
this refers to the cocoapos version here https://github.com/CocoaPods/Specs/blob/cc4fa5b9f9e8e6b22675deb2cb8706dfb2daa08f/Specs/0/3/5/Firebase/6.3.0/Firebase.podspec.json#L24-L27 . The "Firebase/InAppMessagingDisplay is an subspecs of this here https://github.com/CocoaPods/Specs/blob/cc4fa5b9f9e8e6b22675deb2cb8706dfb2daa08f/Specs/0/3/5/Firebase/6.3.0/Firebase.podspec.json#L298-L305